config/ci.rb

```
@@ -0,0 +1,22 @@
+# Run using bin/ci
```

RuboCop (1.85.1)

[convention] Missing frozen string literal comment.

(Style/FrozenStringLiteralComment)